public/resources/remix/index.js

```diff
@@ -6,122 +6,136 @@
 
   var gaTrackingId = 'UA-68630113-1';
 
-  function injectAnalytics($) {
-    var analyticsHtml =
-    '\n<script>\n' +
-    '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' +
-    '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' +
-    'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' +
-    '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' +
-    'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' +
-    'ga(\'send\', \'pageview\');\n' +
-    '</script>\n';
-
-    $(head).append(analyticsHtml);
+  function injectAnalytics() {
+    var analyticsHtml = document.createElement("script");
+    analyticsHtml.type = "text/javascript";
+    analyticsHtml.innerHTML = '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' + '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' + 'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' + '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' + 'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' + 'ga(\'send\', \'pageview\');\n';
+    head.appendChild(analyticsHtml);
   }
 
-  function setupBar(err, $) {
+  function setupBar(err) {
     if(err) {
       console.log("[Thimble Error] Unable to inject Remix UI. Error was: `" + err + "`");
       return;
     }
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
-    var detailsBar = $(".details-bar");
-    detailsBar.attr("style", "");
+    var detailsBar = document.querySelector(".details-bar");
+    detailsBar.setAttribute("style", "");
 
     if(isTouchDevice) {
-      detailsBar.addClass("touch-mode");
-    } else {
-      detailsBar.addClass("mouse-mode");
+      detailsBar.classList.add("touch-mode");
+    }
+    else {
+      detailsBar.classList.add("mouse-mode");
     }
 
-    detailsBar.on("click", ".thimble-button",function(){
-      detailsBar.removeClass("collapsed");
-      return false;
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.className == "thimble-button") {
+        detailsBar.classList.remove("collapsed");
+        return false;
```

Comment:
in HTML5 `return false` gets ignored. If you want to stop propagation you will need to call `event.stopPropagation();` and if you want to prevent browser-default behaviour you'll have to call `event.preventDefault();`.